src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx

```diff
@@ -4,108 +4,2980 @@ tags:
+          </Callout>
+        </TabsPageItem>
+      </TabsPages>
+    </Tabs>
+  </Collapser>
+</CollapserGroup>
+
+## Use custom profiles privately [#private]
+
+The contents of the [kentik/snmp-profiles](https://github.com/kentik/snmp-profiles) repository are loaded to `ktranslate` by default when you build the container or service and are therefore globally applied to all instances.
+
+To support full customization options that allow you to maintain your own collection settings and patterns; you can clone the repository, make changes as you see fit, and load your custom profile directory into your Docker container or Linux service on startup.
+
+The way this is done is by mapping your customized files to the KTranslate instance inside the `etc/ktranslate/profiles/` directory.
+
+Assuming you are in a directory just above `snmp-profiles`, the command to add will look like this:
+
+```shell
+`pwd`/snmp-profiles/profiles:/etc/ktranslate/profiles
+```
+
+For Docker, you would add this as a [volume mount](https://docs.docker.com/storage/volumes/#start-a-container-with-a-volume), which would look similar to this:
+
+```shell
+docker run -d --name ktranslate-$CONTAINER_SERVICE --restart unless-stopped --pull=always -p 162:1620/udp \
+  -v `pwd`/snmp-base.yaml:/snmp-base.yaml \
+  -v `pwd`/snmp-profiles/profiles:/etc/ktranslate/profiles
+  -e NEW_RELIC_API_KEY=$NR_LICENSE_KEY \
+kentik/ktranslate:v2 \
+  -snmp /snmp-base.yaml \
+  -nr_account_id=$NR_ACCOUNT_ID \
+  -metrics=jchf \
+  -tee_logs=true \
+  -service_name=$CONTAINER_SERVICE \
+  -snmp_discovery_on_start=true \
+  -snmp_discovery_min=180 \
+  nr1.snmp
+```
+
+For a Linux service, you will edit the contents of `/etc/ktranslate/profiles` on the local host.
 
 <Callout variant="tip">
-  Be sure to [sync your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) regularly to keep it up to date with changes in the upstream repo.
+  Be sure to pass your custom version of the profiles in every time you launch a container or service to remove the risk of inconsistent behavior in your environment.
 </Callout>
 
-## What kinds of OIDs should I actually include in my profiles? [#oids]
+## Contribute to the public repository [#public]
+
+As an open source repo, contributions are welcome from anyone to create new profiles or improve the existing profiles that will enhance the monitoring capabilities for all users.
+
+New Relic is not involved in the process of reviewing profiles on the third-party repo, all requests for changes and updates must be addressed with Kentik's maintainers via Github issues.
+
+Shortly after a profile is merged new SNMP profiles are automatically available by pulling the new version of the Docker image and launching a new container in your environment. For a Linux service, a new release needs to be created on the [kentik/ktranslate](https://github.com/kentik/ktranslate) repository.
```

Comment:
```suggestion
Shortly after a profile is merged, new SNMP profiles are automatically available. You can pull the new version of the Docker image and launch a new container in your environment. For a Linux service, create a new release on the [kentik/ktranslate](https://github.com/kentik/ktranslate) repository.
```

Reply:
this isn't quite right. the process of creating a new release on the ktranslate repo is a manual process that one of the repo maintainers has to kick off. various customers/users will not be able to do this step.